cnd/src/db/schema.rs

```diff
@@ -104,3 +104,13 @@ table! {
        counterparty -> Text,
    }
 }
+
+table! {
+    finalized_swaps {
+        id -> Integer,
```

Comment:
Do we need a `finalized_at` timestamp? 🤔

Reply:
Yes, timestamp needs to be done _somewhere_.  I think you may be right, finalized is the place for it not created.  Will add.